src/Pages/2DPrinting/2D-printing.css

```diff
@@ -34,6 +34,7 @@
   transform: translateZ(0px);
   animation: fade-in 1s ease-in-out;
   animation-fill-mode: forwards;
+  cursor: pointer;
```

Comment:
nice i learned about the types from you
![image](https://user-images.githubusercontent.com/36345325/89148340-617c0e80-d50e-11ea-8463-5d3d5f2e1a21.png)

Reply:
The front end component still works - I tested it. Stubbed the printingRPC return values. Making PR tonight on SCE-RPC to remove destination parameter.

I need to fix Travis tho

Reply:
so this would be the type of pointer that shows up while in the printing view window?